indexer-opts/meta-table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE meta (
```

Comment:
Let's name it so that everyone understand it's not one of the regular tables?
`__meta`? (pythonista in me)
`meta__indexers`? (if we would need to have other meta info)

Reply:
I will rename it to `__meta` to remind us our Python past 🐍